v2/netconf/server/netconf/server_test.go

```diff
@@ -0,0 +1,89 @@
+package netconf
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/damianoneill/net/v2/netconf/ops"
+
+	"github.com/damianoneill/net/v2/netconf/common"
+	"github.com/damianoneill/net/v2/netconf/server/ssh"
+	xssh "golang.org/x/crypto/ssh"
+
+	assert "github.com/stretchr/testify/require"
+)
+
+// Defines credentials used for test sessions.
+const (
+	TestUserName = "testUser"
+	TestPassword = "testPassword"
+)
+
+var sessionFactory = func(sh *SessionHandler) SessionCallback {
+	fmt.Println("Session", sh.sid, sh.svrcon.Conn.RemoteAddr())
+	return &callback{}
+}
+
+type callback struct{}
+
+func (cb *callback) Capabilities() []string {
+	return common.DefaultCapabilities
+}
+
+func (cb *callback) HandleRequest(req *RpcRequestMessage) *RpcReplyMessage {
+	data := ReplyData{Data: responseFor(req)}
+
+	errors := []common.RPCError{}
+	return &RpcReplyMessage{Data: data, MessageID: req.MessageID,
+		Errors: errors}
+}
+
+func responseFor(req *RpcRequestMessage) string {
+	switch req.Request.XMLName.Local {
+	case "get":
+		return `<top><sub attr="avalue"><child1>cvalue</child1><child2/></sub></top>`
+	case "get-config":
+		return `<top><sub attr="cfgval1"><child1>cfgval2</child1></sub></top>`
+	case "edit-config":
```

Comment:
Not called in test?

Reply:
No - the server framework is not aware of spc operations. This was there for illustration, but I've now commented the edit-config case, with // etc..